client/delegations.go

```diff
@@ -75,7 +75,7 @@ func (c *Client) loadLocalSnapshot() (*data.Snapshot, error) {
 }
 
 // loadDelegatedTargets downloads, decodes, verifies and stores targets
-func (c *Client) loadDelegatedTargets(snapshot *data.Snapshot, role string, verifier verify.DelegationsVerifier) (*data.Targets, error) {
+func (c *Client) loadDelegatedTargets(snapshot *data.Snapshot, role string, db *verify.DB) (*data.Targets, error) {
```

Comment:
can we somehow hide the implementation details here?

Reply:
I think this is a simplification in implementation. Before, `verify.DelegationsVerifier` was essentially the same type as `verify.DB`, but used only in certain contexts.